googletest/include/gtest/gtest.h

```diff
@@ -159,6 +163,10 @@ GTEST_DECLARE_bool_(throw_on_failure);
 // the specified host machine.
 GTEST_DECLARE_string_(stream_result_to);
 
+#if GTEST_USE_OWN_FLAGFILE_FLAG_
```

Comment:
This seems to be completely new (#if GTEST_USE_OWN_FLAGFILE_FLAG_)... also how is this related to clang warnings
thanks again

Reply:
Same thing really, except that in this case the definition depends on `GTEST_USE_OWN_FLAGFILE_FLAG_` (see gtest.cc line 326).
The declaration should be surrounded by the same condition so that you get either declaration *and* definition, or neither.